cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -551,11 +556,6 @@ var (
 				paths = append(paths, "locations")
```

Comment:
Is the location update affected by the fact that we do `SetFromFlags` earlier now ?

Reply:
I don't think so. We [don't parse the locations](https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.25/pkg/ttnpb/end_device_flags.pb.go#L2930) in `SetFromFlags` and `hasUpdateDeviceLocationFlags` also checks if the location flags were explicitly set and then creates the required Go structs.
These two seem indepejdent of each other to me.